client/watch.js

```diff
@@ -375,10 +388,17 @@ Template.watch_page.onRendered(function(){
   this.mainPlayerYTApiActivated = false;
   this.mainPlayerUSApiActivated = false;
   
+  this.checkTime = Meteor.setInterval(()=>{
+    if(mainPlayer && mainPlayer.getElapsedTime){
+      Session.set('currentTimeElapsed', mainPlayer.getElapsedTime());
+    }
+  }.bind(this),4000);
```

Comment:
is the `.bind(this)` necessary? I'm not sure what it does in this context, especially since `this` isn't called from within the function anywhere, and also the `=>` passes through `this` anyway.